lib/ansible/modules/clustering/kubevirt/kubevirt_vm.py

```diff
@@ -376,10 +376,7 @@ def execute_module(self):
                 ret = self.manage_state(state)
                 changed = changed or ret
```

Comment:
We still populate changed param. Why do we want to remove from `exit_json` and leave it in the code?